.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-  - 5.4
```

Comment:
Some dependencies (ie discovery package) requires php 5.5

> On 18 aug. 2016, at 23:03, Florian Voutzinos <notifications@github.com> wrote:
> 
> In .travis.yml <https://github.com/florianv/swap/pull/26#discussion_r75388823>:
> 
> > @@ -1,7 +1,6 @@
> >  language: php
> >  
> >  php:
> > -  - 5.4
> Can we keep PHP 5.4 support ?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/florianv/swap/pull/26/files/80a48d67ee0057b66788b02e15f8a26114046c52#r75388823>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABN1Ri8THG72YngvqxEwpUcjnqax3oA_ks5qhMiegaJpZM4HqSa4>.
>